src/coffea/analysis_tools.py

```diff
@@ -557,18 +1089,36 @@ def require(self, **names):
         returns a boolean array where an entry is True if the corresponding entries
         ``cut1 == True``, ``cut2 == False``, and ``cut3`` arbitrary.
         """
+        for cut, v in names.items():
+            if not isinstance(cut, str) or cut not in self._names:
+                raise ValueError(
+                    "All arguments must be strings that refer to the names of existing selections"
+                )
+
         consider = 0
```

Comment:
you can define `consider` as the appropriate dtype up here, as well as require.

Reply:
That didn't work when I was trying it out. I had to to at the return statement after performing operations with them. It would also fail in eager mode with int64 if I did that.